src/position.cpp

```diff
@@ -1139,8 +1138,8 @@ bool Position::is_draw(int ply) const {
 }
 
 
-// Tests whether there has been at least one repetition
-// of positions since the last capture or pawn move.
+// Checks if there has been at least one position
+// repetition since the last capture or pawn move.
```

Comment:
The term "position repetition" might be ambiguous.

Reply:
I'm not sure about this, I feel that "position repetition" is very clear and correct from a grammatical point.
However, if more devs prefer "repetition of positions" I can revert it.

Reply:
I do prefer using "repetition of positions" instead of "position repetition"

Reply:
Resolved